automation/README.md

```diff
@@ -9,35 +9,39 @@ See [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-au
 
 ### Set Up
 
-Render configs:
+#### Render configs
+
+##### Against dev fiab Sam
+```bash
+./render-local-env.sh
+```
+
+##### Against local UI pointing to dev fiab Sam
 ```bash
-./render-local-env.sh [branch of firecloud-automated-testing] [vault token] [env] [service root]
+LOCAL_UI=true ./render-local-env.sh
+```
+
+##### Against local Sam pointing to the dev live env
```

Comment:
I don't understand this one?  "Local Sam pointing to live dev Sam"?  Do you mean you're running the Sam app locally but using the live dev Sam's database?

Reply:
I initially was thinking of it as local Sam pointing to the live dev Sam's DB, but it should actually just say a local Sam, because the local Sam can point to either dev DB or a local DB. Thanks for the catch -- updated.